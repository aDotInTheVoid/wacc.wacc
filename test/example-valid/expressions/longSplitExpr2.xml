<unit>
  <main>
    <addr_local>0</addr_local>
    <num>1</num>
    <num>2</num>
    <op>add</op>
    <num>3</num>
    <op>add</op>
    <num>4</num>
    <op>add</op>
    <num>5</num>
    <op>add</op>
    <num>6</num>
    <op>add</op>
    <num>7</num>
    <op>add</op>
    <num>8</num>
    <op>add</op>
    <num>9</num>
    <op>add</op>
    <num>10</num>
    <op>add</op>
    <num>11</num>
    <op>add</op>
    <num>12</num>
    <op>add</op>
    <num>13</num>
    <op>add</op>
    <num>14</num>
    <op>add</op>
    <num>15</num>
    <op>add</op>
    <num>16</num>
    <op>add</op>
    <num>17</num>
    <op>add</op>
    <assign_do/>
    <addr_local>1</addr_local>
    <num>1</num>
    <neg/>
    <num>2</num>
    <op>sub</op>
    <num>3</num>
    <op>sub</op>
    <num>4</num>
    <op>sub</op>
    <num>5</num>
    <op>sub</op>
    <num>6</num>
    <op>sub</op>
    <num>7</num>
    <op>sub</op>
    <num>8</num>
    <op>sub</op>
    <num>9</num>
    <op>sub</op>
    <num>10</num>
    <op>sub</op>
    <num>11</num>
    <op>sub</op>
    <num>12</num>
    <op>sub</op>
    <num>13</num>
    <op>sub</op>
    <num>14</num>
    <op>sub</op>
    <num>15</num>
    <op>sub</op>
    <num>16</num>
    <op>sub</op>
    <num>17</num>
    <op>sub</op>
    <assign_do/>
    <addr_local>2</addr_local>
    <num>1</num>
    <num>2</num>
    <op>mul</op>
    <num>3</num>
    <op>mul</op>
    <num>4</num>
    <op>mul</op>
    <num>5</num>
    <op>mul</op>
    <num>6</num>
    <op>mul</op>
    <num>7</num>
    <op>mul</op>
    <num>8</num>
    <op>mul</op>
    <num>9</num>
    <op>mul</op>
    <num>10</num>
    <op>mul</op>
    <assign_do/>
    <addr_local>3</addr_local>
    <num>10</num>
    <assign_do/>
    <local>0</local>
    <local>1</local>
    <op>add</op>
    <local>2</local>
    <local>3</local>
    <op>div</op>
    <op>add</op>
    <print><kind>i32</kind><newline>true</newline></print>
    <local>0</local>
    <local>1</local>
    <op>add</op>
    <local>2</local>
    <local>3</local>
    <op>div</op>
    <op>add</op>
    <num>256</num>
    <op>mod</op>
    <print><kind>i32</kind><newline>true</newline></print>
    <local>0</local>
    <local>1</local>
    <op>add</op>
    <local>2</local>
    <local>3</local>
    <op>div</op>
    <op>add</op>
    <exit/>
  </main>
</unit>