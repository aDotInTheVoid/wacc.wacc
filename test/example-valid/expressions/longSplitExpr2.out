362880
128


---
Exit: 128
